.env.example

```
@@ -23,4 +23,9 @@ MINIO_SECURE=false
 MINIO_PRESIGNED_EXPIRES_SECONDS=3600
 
 CELERY_BROKER_URL=redis://redis:6379/0
-CELERY_RESULT_BACKEND=redis://redis:6379/1
\ No newline at end of file
+CELERY_RESULT_BACKEND=redis://redis:6379/1
+
+VLLM_BASE_URL = "https://llm.example.ru/v1"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)